ansible/layers-service.yml

```diff
@@ -0,0 +1,11 @@
+- hosts: layers-service
+  roles:
+    - java
+    - common
+    - tomcat
+    - apache
+    - geoserver
+    - postgresql    
+    - { role: postgresql_instance, db_name: "{{ layers_db_name | default('layersdb')}}",  db_username: "{{ layers_db_username | default('postgres') }}", db_password: "{{ layers_db_password | default('password') }}" }
```

Comment:
Can the db configuration be moved into the layers-service role to be consistent with other playbooks?